# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dvorakng/install_path.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-04-25 23:52:48.000000000 +0200
+++ ./Makefile	2003-08-01 11:19:27.000000000 +0200
@@ -1,4 +1,4 @@
-INSTALL = /usr/local/games/
+INSTALL = $(prefix)/bin/
 
 # Uncomment these lines if you want to change the defaults to something else
 # (This is/will be removed - and better mechanism will be used for this purpose - Nopik)
